components/mailbox/src/index.html

```diff
@@ -49,7 +49,7 @@
         const composer = document.querySelector("nylas-composer");
         composer.show_header = true;
         composer.show_minimize_button = false;
-        composer.show_from = false;
+        composer.show_from = true;
```

Comment:
Do we need to make this change?

Reply:
We don't really need it as it is only for the demo.